pkg/tcpip/transport/tcp/cubic.go

```diff
@@ -18,9 +18,53 @@ import (
 	"math"
 	"time"
 
+	"gvisor.dev/gvisor/pkg/tcpip"
 	"gvisor.dev/gvisor/pkg/tcpip/stack"
 )
 
+// effectivelyInfinity is an initialization value used for round-trip times
+// that are then set using minDuration.  It is equal to approximately 100
+// years: large enough that it will always be greater than a real TCP
+// round-trip time, and small enough that it fits in time.Duration.
+const effectivelyInfinity = 876000 * time.Hour
+
+// c.f. RFC 9406 Section 4.3.  RTT = round-trip time.
+const (
+	// The delay increase sensitivity is determined by minRTTThresh and
+	// maxRTTThresh. Smaller values of minRTTThresh may cause spurious exits
+	// from slow start. Larger values of maxRTTThresh may result in slow start
+	// not exiting until loss is encountered for connections on large RTT paths.
+	minRTTThresh = 4 * time.Millisecond
+	maxRTTThresh = 16 * time.Millisecond
+
+	// minRTTDivisor is a fraction of RTT to compute the delay threshold. A
+	// smaller value would mean a larger threshold and thus less sensitivity to
+	// delay increase, and vice versa.
+	minRTTDivisor = 8
+
+	// nRTTSample is the minimum number of RTT samples in the round before
+	// considering whether to exit the round due to increased RTT.
+	nRTTSample = 8
+
+	// ackDelta is the maximum time between ACKs for them to be considered part
+	// of the same ACK Train during HyStart
+	ackDelta = 2 * time.Millisecond
+)
```

Comment:
Just to be clear: this PR doesn't have conservative slow start, right? The only bits taken from HyStart++ are the constants?

Reply:
That's right.  I think I'm going to drop the reference to RFC 9406 here.  Some of the comments come from that RFC, e.g. "A smaller value would mean a larger threshold and thus less sensitivity to delay increase, and vice versa." But, I think they are understandable on their own, and since we don't implement the RFC 9406 variant of HyStart, it will be less confusing.